ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/util/CommandLineParserAssertions.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.ethsigner.CmdlineHelpers.removeFieldFrom;
+
+import tech.pegasys.ethsigner.CommandlineParser;
+
+import java.io.ByteArrayOutputStream;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public interface CommandLineParserAssertions {
```

Comment:
nit: why an interface here? It's not being used as util class just using the the static method not as an interface

Reply:
At the moment it contains static methods in it. I find it easier to define static helper methods in an interface as you can either implement this interface to inherit the methods or static import them. The other option is to declare it as a final class.

Reply:
Refactored it to a `final class` instead.

Reply:
Not too bothered by this, though I would have gone with util class using a private constructor.